.goreleaser.yml

```diff
@@ -28,7 +28,52 @@ builds:
       # - arm
       - arm64
     main: ./cmd/osv-scanner/main.go
-    binary: '{{ .ProjectName }}_v{{ .Version }}'
+
+dockers:
+  - image_templates:
+      - 'ghcr.io/google/osv-scanner:{{ .Tag }}-amd64'
+    dockerfile: goreleaser.dockerfile
+    use: buildx
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.title=osv-scanner"
+      - "--label=org.opencontainers.image.description=Vulnerability scanner written in Go which uses the data provided by https://osv.dev"
+      - "--label=org.opencontainers.image.licenses=Apache License 2.0"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.source={{.GitURL}}"
+      - "--label=org.opencontainers.image.url={{.GitURL}}"
+      - "--platform=linux/amd64"
+  - image_templates:
```

Comment:
```suggestion
  # Arch: arm64
  - image_templates:
```
Just to make it clear the arch is what's different about these two builds, arm64 and amd64 looks very similar.